lib/jekyll/commands/new_theme.rb

```diff
@@ -22,7 +26,8 @@ def process(args, opts)
         raise Jekyll::Errors::InvalidThemeName, "You must specify a theme name."
       end
 
-      new_theme_name = args.join("_")
+      new_theme_name = args.first
+      opts["theme_path"] = args[1..-1].join(" ") if args.size > 1
```

Comment:
suppose I enter **Long Island** as the theme_name.. how would this translate as the .gemspec filename?
**long_island.gemspec** or **long island.gemspec**?